src/main/kotlin/calculator/Calculator.kt

```diff
@@ -1,25 +1,29 @@
 package calculator
 
 class Calculator {
-    fun calculate(input: String?): Int {
-        if (input.isNullOrBlank()) {
-            throw IllegalArgumentException("입력값이 null이거나 빈 공백 문자입니다.")
+    fun run(input: String?): Int {
+        require(input.isNullOrBlank()) {
+            "입력값이 null이거나 빈 공백 문자입니다."
         }
 
-        val inputs = input.split(" ")
+        val inputs = input?.split(" ")
```

Comment:
```suggestion
        val inputs = input?.split(" ") ?: emptyList()
```
최대한 위에서 nullable을 제거해주면 아래에서 반복적으로 재처리할 필요가 없어집니다.